app/models/miq_schedule/import_export.rb

```diff
@@ -57,11 +57,12 @@ def handle_miq_report_attributes_for_import(miq_schedule)
       miq_schedule
     end
 
-    def import_from_hash(miq_schedule, _options = nil)
+    def import_from_hash(miq_schedule, options = nil)
       miq_schedule = handle_miq_report_attributes_for_import(miq_schedule) if miq_schedule["resource_type"] == "MiqReport"
 
-      unless miq_schedule['userid'] == 'admin' || miq_schedule['userid'] == 'system'
-        miq_schedule['userid'] = User.find_by(:id => miq_schedule['userid'])
+      if options[:userid] || miq_schedule[:userid]
```

Comment:
Doesn't this need to be a string? 'userid' and the next line probably too?

Reply:
Also I think options[:userid] needs a presence check so it doesn't fail on nil?

Reply:
@d-m-u, you are right, good 👀  I fixed, thanks!

Reply:
@d-m-u 👏